docs/docs/guide/account.md

```diff
@@ -0,0 +1,20 @@
+---
+id: account
+title: Account & Network Settings
+slug: /account
+---
+
+## Adding a new account
+
+In order to use another account that is derived from your seed phrase to Freighter, you can do the following:
+
+- From the account balances page, you can click on the "accounts" dropdown from the identicon in the upper left corner of the screen in order to open the account options.
+- From the options page, you can "Import a Stellar secret key" if you already have your secret key or you can "Create new Stellar wallet" if you want to generate one.
```

Comment:
It's probably important to note here that if you import by S key it won't be tied to your seed phrase

Reply:
Yeah good call, updated in c4ffd8dbf43c606e3741e1ca7592b28de7bb60cf